src/gui/qgsfieldexpressionwidget.h

```diff
@@ -244,7 +250,7 @@ class GUI_EXPORT QgsFieldExpressionWidget : public QWidget
     QToolButton *mButton = nullptr;
     QgsFieldProxyModel *mFieldProxyModel = nullptr;
     QString mExpressionDialogTitle;
-    std::shared_ptr<const QgsDistanceArea> mDa;
+    std::shared_ptr<const QgsDistanceArea> mDistanceArea;
```

Comment:
I realise you didn't write this -- but there's no reason for this to be a shared ptr. A unique_ptr would better express the situation here.

Reply:
maybe in a separate PR